pkgs/os-specific/linux/fxload/default.nix

```diff
@@ -2,21 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "fxload";
-  version = "2002.04.11";
+  version = "unstable-2013-01-03";
 
   src = fetchurl {
-    url = "mirror://sourceforge/linux-hotplug/fxload-${lib.replaceStrings ["."] ["_"] version}.tar.gz";
-    sha256 = "1hql93bp3dxrv1p67nc63xsbqwljyynm997ysldrc3n9ifi6s48m";
+    url = "mirror://sourceforge/fx3load/fx3load.tgz";
```

Comment:
Please upload that to web.archive.org

Reply:
Why?

Reply:
Even more a reason. The URL is not versioned and it could be removed in the future if it is pretty dead.

Reply:
That's fair.

Reply:
@spacekitteh Can you move back to `fetchgit`? Probably better than tossing it up on IA.